### PR TITLE
fix #710

### LINKED ISSUE #710: Automatically wrap code block lines when printing or exporting as PDF
### Description

If you export or print a document with long code block lines, the lines are not wrapped and the scrollbar is printed too.

- [x] Can you reproduce the issue? 

### Steps to reproduce

1. Export the following document without `\` and open the PDF:
```
\```
0123456789ABCDEF 0123456789ABCDEF 0123456789ABCDEF 0123456789ABCDEF 0123456789ABCDEF 0123456789ABCDEF 0123456789ABCDEF 0123456789ABCDEF

0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF0123456789ABCDEF
\```
```

**Expected behavior:**

All long lines should be wrapped to the next line.

**Actual behavior:**

mt_exportwordwrapping

### Versions

- Mark Text: v0.13.65
- Operating system: all

### PR DESCRIPTION
**Pull request by Jocs**
| Q                | A
| ---------------- | ---
| Fixed tickets    | #710 
| License          | MIT

### Description

[Description of the bug or feature]

--

### CODE REVIEW

**Comment by Jocs** [comment]
@fxha I'll update change log after all these bug fix finished. :smile:

File: src/muya/lib/utils/exportHtml.js [hunk 1]
```diff
@@ -88,6 +88,10 @@ class ExportHtml {
       margin-top: 0;
       display: inline-block;
     }
+    .markdown-body pre code.fenced-code-block {
```
Comment (fxha) [inline src/muya/lib/utils/exportHtml.js]: Would it not be better, if the text is only wrapped when exported as PDF or printed? Meaning wrap your added code inside `@media print`.

**Comment by Jocs** [comment]
Need merged?

**Comment by fxha** [comment]
Yes, if you agree with my changes :+1:

**Pull request merged**